easybuild/easyblocks/generic/configuremake.py

```diff
@@ -175,6 +176,7 @@ def extra_options(extra_vars=None):
         extra_vars = EasyBlock.extra_options(extra=extra_vars)
         extra_vars.update({
             'build_cmd': [DEFAULT_BUILD_CMD, "Build command to use", CUSTOM],
+            'build_targets': [DEFAULT_BUILD_TARGETS, "Targets to build", CUSTOM],
```

Comment:
This should say "`Target name (string) or list of target names to build`"?

Reply:
done